The lines starting with _ should get translated, but the build system lacks support for it.

--- Build/bin/Rodent.desktop.in
+++ Build/bin/Rodent.desktop.in
@@ -1,13 +1,12 @@
 [Desktop Entry]
 Version=1.0
 Type=Application
-Encoding=UTF-8
-_Name=Rodent
-_GenericName=Rodent Filemanager
-_Comment=Advanced user filemanager
+Name=Rodent
+GenericName=Rodent Filemanager
+Comment=Advanced user filemanager
 TryExec=rodent
 Exec=rodent
 Icon=Rodent-system
 Terminal=false
-Categories=Core;Filemanager;
+Categories=System;Utility;Core;GTK;FileTools;FileManager;
 StartupNotify=false
--- Build/bin/Rodent-diff.desktop.in
+++ Build/bin/Rodent-diff.desktop.in
@@ -1,13 +1,12 @@
 [Desktop Entry]
 Version=1.0
 Type=Application
-Encoding=UTF-8
-_Name=Rodent Differences tool
-_GenericName=Rodent Differences tool
-_Comment=File difference and patch visualiser
+Name=Rodent Differences tool
+GenericName=Rodent Differences tool
+Comment=File difference and patch visualiser
 TryExec=rodent-diff
 Exec=rodent-diff
 Icon=diff-icon
 Terminal=false
-Categories=Core;Utility;
+Categories=Utility;Core;GTK;FileTools;
 StartupNotify=false
